### PR TITLE
update rust bindings to include scanner and queries

### PR DESCRIPTION
**Pull request by the-mikedavis**
connects #29

This fixes `cargo test` runs and includes the highlight queries in the rust package.

### CODE REVIEW

File: bindings/rust/lib.rs [hunk 1]
```diff
@@ -35,7 +35,7 @@ pub const NODE_TYPES: &'static str = include_str!("../../src/node-types.json");
 
 // Uncomment these to include any queries that this grammar contains
 
-// pub const HIGHLIGHTS_QUERY: &'static str = include_str!("../../queries/highlights.scm");
+pub const HIGHLIGHTS_QUERY: &'static str = include_str!("../../queries/highlights.scm");
 // pub const INJECTIONS_QUERY: &'static str = include_str!("../../queries/injections.scm");
 // pub const LOCALS_QUERY: &'static str = include_str!("../../queries/locals.scm");
 // pub const TAGS_QUERY: &'static str = include_str!("../../queries/tags.scm");
```
Comment (jonatanklosko) [inline bindings/rust/lib.rs:41]: Should we expose this as well?

```suggestion
pub const TAGS_QUERY: &'static str = include_str!("../../queries/tags.scm");
```

**Review by jonatanklosko — APPROVED** [review]
Awesome, thanks for taking care of this!!

File: bindings/rust/lib.rs (hunk 1 above)
Comment (the-mikedavis) [reply bindings/rust/lib.rs:41]: Whoops how could I forget about these 😅

**Comment by the-mikedavis** [comment]
I think it's worth discussing whether tree-sitter-elixir should be published to crates.io and/or npm. Some grammars are published (for example [ruby](https://crates.io/crates/tree-sitter-ruby)) but many are not, and those that are do not usually get updated frequently. Most of the larger tree-sitter consumers I know take dependencies on grammars by git (either by submodules or shallow-clones), and you can do that with `cargo` and `npm`, so IMO there isn't really any benefit in publishing them.

**Comment by jonatanklosko** [comment]
@the-mikedavis I agree with everything you said, using Git dependencies sounds fine, so let's wait until there's a clear need for it being published :)

**Pull request merged**